Classes/Neos/Neos/Ui/Domain/Model/Changes/AbstractCreate.php

```diff
@@ -176,6 +213,20 @@ protected function createNode(NodeInterface $parent)
             $node->setProperty($key, $value);
         }
 
+        if ($nodeType->isOfType('TYPO3.Neos:Content') && $this->getReferenceData()['subject']['fusionPath']) {
+            $renderNode = new RenderNode();
```

Comment:
...AddToContentCollectionOutOfBand is maybe even better; as the code in JS makes assumptions this is only working in Content Collections